rel-eng/openshift-node.sysconfig

```diff
@@ -1,2 +1,5 @@
 ROLE="node"
 OPTIONS="--loglevel=0"
+# If running node on a separate host please rsync the admin directory ie:
+# rsync root@openshift-master:/var/lib/openshift/openshift.local.certificates/admin /var/lib/openshift/openshift.local.certificates/
+KUBECONFIG="/var/lib/openshift/openshift.local.certificates/admin/.kubeconfig"
```

Comment:
We are delaying creating a formal config until the set of required options stabilize.  We will support env vars before we support more flags, for sure.